doc/source/examples/incompressible-flow/2d-transient-flow-around-cylinder/2d-transient-flow-around-cylinder.rst

```diff
@@ -0,0 +1,266 @@
+                set Function expression = 0
+            end
+        end
+        subsection bc 2
+            set type              = slip
+        end
+    end
+
+Physical Properties
+~~~~~~~~~~~~~~~~~~~
+
+The Reynolds number must be high enough to capture a transient flow and study the evolution of the drag and lift coefficients in time. Therefore, we set Re = 200 through the value of the kinematic viscosity in the same manner as for the `2D Lid-driven cavity flow <https://lethe-cfd.github.io/lethe/examples/incompressible-flow/2d-lid%E2%80%90driven-cavity-flow/lid%E2%80%90driven-cavity-flow.html>`_. Since :math:`U_\infty = 1` and the :math:`D = 1`, we have :math:`Re=\frac{1}{\nu}`, where :math:`\nu` is the kinematic viscosity.
+
+.. code-block:: text
+
+    subsection physical properties
+        subsection fluid 0
+          set kinematic viscosity            = 0.005
+        end
+    end
+
+Forces
+~~~~~~
+
+Since we want to study the time evolution of the drag and lift coefficients, the force acting on the boundaries must be computed. We thus use the ``forces`` subsection:
+
+.. code-block:: text
+
+   subsection forces
+      set verbosity             = verbose
+      set calculate force       = true
+      set calculate torque      = false
+      set force name            = force
+      set output precision      = 10
+      set calculation frequency = 1
+      set output frequency      = 1
+   end
+
+As we set ``calculation frequency`` to 1, the forces on each boundary are computed at each time step and written in the file specified by the field ``force name``.
+
+.. note::
+
+  The drag and lift coefficients are obtained with the forces acting on the wall of the cylinder (i.e., ``f_x`` and ``f_y``  written in the file ``forces.00.dat``) :
+
+  .. math::
+
+    C_D = \frac{2 f_x}{\rho U_\infty^2 D} \text{ and } C_L = \frac{2 f_y}{\rho U_\infty^2 D}
+
+  where :math:`\rho = 1`. This way, we can obtain the evolution in time of both coefficients.
+
+.. warning::
+
+  The computational cost of writing this output file at each time step by setting ``output frequency`` to 1 can be significant, as explained in `Force and torque calculation <https://lethe-cfd.github.io/lethe/parameters/cfd/force_and_torque.html>`_. However, in the present example, the computational cost is still fairly low so ``output frequency = 1`` is used.
+
+
+Running the simulation
+----------------------
+The simulation is launched in parallel using 10 CPUs, as explained in `2D Transient flow around an Ahmed body <https://lethe-cfd.github.io/lethe/examples/incompressible-flow/2d-transient-around-ahmed-body/2d-transient-around-ahmed-body.html>`_ :
+
+.. code-block:: text
+
+  mpirun -np 10 gls_navier_stokes_2d cylinder.prm
+
+.. warning::
+
+  The estimated time to simulate 200 seconds is about 2 hours 5 minutes with 10 CPUs.
+
+Results
+-------
+
+The time evolution of the drag and lift coefficients is obtained from a Gnuplot script available in the example folder:
```

Comment:
Can you mention how to call this script and the post-processing script?

Reply:
It's added :)